package com.gut;
//selection sort
public class Selectionsort {

	public static void main(String[] args) {
		int[] a = { 16, 25, 12, 22, 11 };
		for (int i = 0; i < a.length - 1; i++) {
			int min = i;
			for (int j = i + 1; j < a.length; j++) {
				if (a[j] < a[min]) {
					min = j;
				}
			}
			int temp = a[i];
			a[i] = a[min];
			a[min] = temp;
		}
		for (int i = 0; i < a.length; i++) {
			System.out.print(a[i]+"\t");
		}
	}

}
